anitya/tests/test_flask_api_v2.py

```diff
@@ -35,6 +35,475 @@ def _read_json(output):
     return json.loads(output.get_data(as_text=True))
 
 
+class PackagesResourceGetTests(DatabaseTestCase):
+    """Tests for HTTP GET on the ``api/v2/packages/`` resource."""
+
+    def setUp(self):
+        super(PackagesResourceGetTests, self).setUp()
+        self.app = self.flask_app.test_client()
+        session = Session()
+        self.user = models.User(email='user@example.com', username='user')
+        self.api_token = models.ApiToken(user=self.user)
+        fedora = models.Distro('Fedora')
+        debian = models.Distro('Debian')
+        jcline_linux = models.Distro('jcline linux')
+        session.add_all([self.user, self.api_token, fedora, debian, jcline_linux])
+        session.commit()
+
+    def test_no_packages(self):
+        """Assert querying packages works, even if there are no packages."""
+        output = self.app.get('/api/v2/packages/')
+        self.assertEqual(output.status_code, 200)
+        data = _read_json(output)
+
+        self.assertEqual(data, {'page': 1, 'items_per_page': 25, 'total_items': 0, 'items': []})
+
+    def test_authenticated(self):
+        """Assert the API works when authenticated."""
```

Comment:
Should there also be a ```test_unauthenticated```?

Reply:
The rest of the tests don't use authentication so it's tested in the basic success test